noxfiles/git_nox.py

```diff
@@ -41,13 +41,26 @@ def get_all_tags(repo):
     return sorted(repo.tags, key=lambda t: t.commit.committed_datetime, reverse=True)
 
 
-def get_current_tag() -> Optional[str]:
+def get_current_tag(
```

Comment:
mega-nit (please ignore): this change makes a `get_*` function potentially create a tag, so I'd name it `get_or_create_current_tag` 😄

Reply:
well my mega-nit-response is that my change doesn't do that, it was always creating a tag. my change here allows it to fetch a pre-existing tag. 😄

i agree the function could be named better, but with that in mind, i'm gonna leave it as is for now, just don't want to touch _too_ much right before we release, and this function is used in a lot of places within our nox codepaths.